bgpd/bgpd.c

```diff
@@ -5121,6 +5124,28 @@ void peer_port_unset(struct peer *peer)
 	peer->port = BGP_PORT_DEFAULT;
 }
 
+/* Set the TCP-MSS value in the peer structure,
+ * This gets applied only after connection reset
+ * So this value will be used in bgp_connect.
+ */
+int peer_tcp_mss_set(struct peer *peer, uint32_t tcp_mss)
```

Comment:
I see this function is not checked anywhere, can be just void?

Reply:
Done